docs/debugging.md

```diff
@@ -0,0 +1,305 @@
+[2018-01-22T21:23:10.429Z] [DEBUG] - Batch idx [ 9 ] -> [ 135bd0df0401e2fdd52fd136935014fb ]
+[2018-01-22T21:23:10.429Z] [DEBUG] - Batch idx [ 10 ] -> [ 67042296c7c95e84142f21f58da2ebfe ]
+[2018-01-22T21:23:10.429Z] [DEBUG] - Batch idx [ 11 ] -> [ 0e991006d21029e47abe71acc255e807 ]
+[2018-01-22T21:23:10.429Z] [DEBUG] - Batch idx [ 12 ] -> [ ca91b61da8476984f18fc13883ae2fdb ]
+[2018-01-22T21:23:10.429Z] [DEBUG] - Batch idx [ 13 ] -> [ b43a4272a6efcaaa3e0b9616324f1099 ]
+[2018-01-22T21:23:10.429Z] [DEBUG] - Batch idx [ 14 ] -> [ 693cb128e68365830c913631300deac0 ]
+[2018-01-22T21:23:10.43Z] [DEBUG] - Batch idx [ 15 ] -> [ c65fbd4e701cb71d74fd2cc35e14432b ]
+[2018-01-22T21:23:10.43Z] [DEBUG] - Batch idx [ 16 ] -> [ ab24ffd54da0aefdea5277e0edce8425 ]
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secret parameters out of specs...
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-wordpress-ha-apb
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-rocketchat-apb
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-mediawiki-apb
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-hello-world-apb
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-postgresql-apb
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-jenkins-apb
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-manageiq-apb
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-mysql-apb
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-thelounge-apb
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-nginx-apb
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-mariadb-apb
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-pyzip-demo-apb
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-etherpad-apb
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-hello-world-db-apb
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-pyzip-demo-db-apb
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-rds-postgres-apb
+[2018-01-22T21:23:10.43Z] [DEBUG] - Filtering secrets from spec dh-hastebin-apb
+172.17.0.4 - - [22/Jan/2018:21:23:10 +0000] "GET /ansible-service-broker/v2/catalog HTTP/1.1" 200 79226
+[2018-01-22T21:32:17.141Z] [INFO] - Broker configured to refresh specs every 10m0s seconds
+```
+
+If you see connection errors in the logs, review the [Service Catalog and ASB Communication Troubleshooting Guide](https://github.com/openshift/ansible-service-broker/blob/master/docs/troubleshooting.md#service-catalog-and-broker-communication-issues) for further troubleshooting steps.
+
+## Debugging APBs
+
+Once the ASB is communication with the service catalog have been verified, you may view information about the APBs. For example, you may want to get a list of all of your APBs (services), along with all of their service plans.
```

Comment:
s/Once the ASB is communication with the service catalog have been verified/Once the ASB's communication with the service catalog has been verified

Reply:
thanks, will update